libcodechecker/analyze/log_parser.py

```diff
@@ -69,16 +69,55 @@ def parse_compiler_includes(lines):
             # /System/Library/Frameworks (framework directory)
             fpos = line.find("(framework directory)")
             if fpos == -1:
-                include_paths.append("-isystem " + line)
+                include_paths.append(line)
             else:
-                include_paths.append("-isystem " + line[0:fpos-1])
+                include_paths.append(line[0:fpos-1])
 
         if line.startswith(start_mark):
             do_append = True
 
     return include_paths
 
 
+def filter_compiler_includes(include_dirs):
+    """
+    Filter the list of compiler includes.
+    We want to elide e.g. GCC's include-fixed directory.
```

Comment:
Maybe you could refer to the documentation about the include-fixed directories (that is about to be commited).